tests/bluetooth/tester/src/btp_cap.c

```diff
@@ -0,0 +1,1053 @@
+/* btp_cap.c - Bluetooth CAP Tester */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/audio/cap.h>
+
+#include "btp/btp.h"
+#include "btp_bap_audio_stream.h"
+#include "bap_endpoint.h"
+#include <zephyr/bluetooth/audio/csip.h>
+#include "zephyr/sys/byteorder.h"
+#include <stdint.h>
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_cap
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
+
+#include "btp_bap_unicast.h"
+#include "btp_bap_broadcast.h"
+
+static const struct bt_csip_set_coordinator_set_member *set_members[CONFIG_BT_MAX_CONN];
+struct bt_csip_set_coordinator_csis_inst *cur_csis_inst;
+
+static inline struct bt_bap_stream *stream_unicast_to_bap(struct btp_bap_unicast_stream *stream)
+{
+	return &stream->audio_stream.cap_stream.bap_stream;
+}
+
+static inline struct bt_cap_stream *stream_unicast_to_cap(struct btp_bap_unicast_stream *stream)
+{
+	return &stream->audio_stream.cap_stream;
+}
+
+static inline struct bt_cap_stream *stream_broadcast_to_cap(struct btp_bap_broadcast_stream *stream)
+{
+	return &stream->audio_stream.cap_stream;
+}
```

Comment:
Usually there are no reasons to inline functions

```suggestion
static struct bt_bap_stream *stream_unicast_to_bap(struct btp_bap_unicast_stream *stream)
{
	return &stream->audio_stream.cap_stream.bap_stream;
}

static struct bt_cap_stream *stream_unicast_to_cap(struct btp_bap_unicast_stream *stream)
{
	return &stream->audio_stream.cap_stream;
}

static struct bt_cap_stream *stream_broadcast_to_cap(struct btp_bap_broadcast_stream *stream)
{
	return &stream->audio_stream.cap_stream;
}
```